packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -435,7 +439,11 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
             },
             this._onRenderContainer
           )}
-        {errorMessage && <div className={this._classNames.errorMessage}>{errorMessage}</div>}
+        {errorMessage !== undefined && (
+          <div role="region" aria-live="polite" aria-atomic="true" id={this._id + '-error'} className={this._classNames.errorMessage}>
```

Comment:
`this._id + '-error'` should be either in a const or a separate function so it is consistent.